ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -25,12 +35,25 @@ type OwnProps = {
   navigation: NavigationScreenProp<NavigationState>;
 };
 
-type Props = ReturnType<typeof mapStateToProps> & ReduxProps & OwnProps;
+type Props = ReturnType<typeof mapStateToProps> &
+  ReduxProps &
```

Comment:
ReduxProp is not injected.

Reply:
also shouldn't be needed if you dispatch actions with `mapStateToProps` props

Reply:
It will be removed on next commit. Thanks ;)